demo/js/modulr.js

```diff
@@ -898,8 +898,33 @@ var Modulr = (function(window, app){
             }
 
             // shim export
-            function getShimExport(scope) {
-                 return window[scope.split('.')[0]];
+            function getShimExport(id, scope) {
```

Comment:
**TODO:** Set up a config property to turn this on/off - so that users can update to `v3.0.0` without forcing them to change code until they're ready.

```
shim: {
    src: ' . . . ',
    exports: 'foobar',
    traverse: [true/false]
}
```

If traverse is `true` use the new shim exporter. `false` by default if not defined.